### PR TITLE
fix(nfts): allow for big eth amounts

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)

Converting to full string since BigNumber forces scientific notation, but opensea doesn't want
(ex. auto converts to 1e27 when we actually want '1000000000000000000000000000')
 
basePrice: BigNumber | string
extra: BigNumber | string

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts [hunk 1]
```diff
@@ -1065,7 +1065,7 @@ export async function _makeSellOrder({
   quantity: number
   startAmount: number
   waitForHighestBid: boolean
-}): Promise<UnhashedOrder> {
+}): Promise<any> {
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts:1068]: why the change to any?

**Comment by plondon** [comment]
Does this also fix the form issue? When entering 9999?

File: packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts (hunk 1 above)
Comment (rsteubs-bc) [reply packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts:1068]: This is just for testing, because we're going from BigNumber to string, in unhashed order the baseprice type is a BigNumber
